### ISSUE TITLE
Discussion of readings for session 4: Digital collections

### ISSUE THREAD

**Issue by gabrielbodard** [SunoikisisDC/SunoikisisDC-2021-2022#21]
* Clark L., Rossi G.C., Wisdom S. 2020. “Archiving Interactive Narratives at the British Library.” In: Bosser AG., Millard D.E., Hargood C. (eds) _Interactive Storytelling_. ICIDS 2020. _Lecture Notes in Computer Science_, vol 12497. Springer, Cham. Available: https://bl.iro.bl.uk/concern/conference_items/ede94ef4-95b4-4bc7-bce0-b36343cb3542?locale=en
* S. Münster, F. I. Apollonio et al. 2019. “Digital Cultural Heritage meets Digital Humanities.” _International Archives of the Photogrammetry, Remote Sensing and Spatial Information Sciences_ 42.2. Available: https://cris.unibo.it/retrieve/handle/11585/715071/528139/isprs-archives-XLII-2-W15-813-2019.pdf

**Comment by Ghilaevansky** [comment]
The Clark article was an interesting read because it explained the web archives of museums. These collections of UK publications are also used as a safety mechanism, against the risk of disappearance, which is extremely important and useful not to use such valuable works. The online versions also provide accessibility for a wider audience around the world. This was specifically described through the 'Emerging Formats Project', as mentioned in the article. "While large scale collection of such works is unlikely due to their labour-intensive nature, it is possible that in the future further crowd-sourcing activities may be possible, an approach recommended for archiving complex resources in the Digital Curation Centre’s State of the Art report." The Interactive Narratives Collection contains 200 websites and exhibition interaction patterns. It is evident from this article that the world is changing as we know it and it is adapting to the ever evolving media landscape.

**Comment by ellie919** [comment]
The Münster article addresses the similarities and differences between digital heritage and digital humanities. Digital heritage focuses on preservation, education & research, whereas digital humanities focuses on application of digital technologies to support research. The article looks at how research is carried out in these disciplines, what the applications of this research are and problems which can arise in both fields. Since there is no standardised process or methodology for digitising cultural heritage, there can arise issues with carrying out large-scale research across different data sets, especially when using algorithm methods. 
Digital heritage is making digital humanities more accessible to a larger audience, in part by linking space and time. Spatialisation was previously used for urban history, for example, but now digitisation has expanded this practice by enabling researchers to show different changes or actions in an urban space over time. This is an innovative way of thinking about space and time as the two concepts were previously considered separately. 
Digitised images can take the form of primary sources, representation of historical sources, and visualisation in the form of reconstruction of lost heritage objects but these categories can become blurred. 
There are many problems with digitisation of images: not everyone can access them and sometimes museums and other institutions are unwilling to take on digitisation; there are often copyright issues and problems with lack of standardised methodologies; and finally, it requires expertise in many different disciplines to create visual digital resources. 
The article decides that 3D imaging as a research technology is not currently used to its full potential and it can be useful for reproduction and greater understanding of objects. 
Digital humanities has created greater access to digital archives and new tools for research which depends on the quality as well as quantity of the data collected. Reconstruction can be used for non-destructive research but issues can arise such as indicating where an artefact has been digitised from known qualities of an object and where missing qualities of that object have been reconstructed through hypothesis. 
The article concludes that digital cultural heritage involves more textual and spatial resources than digital humanities but less humanities driven research and finally touches on the question of whether this discipline is an art or a science as requires expertise in both fields.

**Comment by Avieava** [comment]
I was thinking about accessibility, of course it is amazing how the digitalisation of artefacts allows information to transgress global boundaries. However on the flip side does anyone have any thoughts on the digital era potentially leading to the death of authentic experiences? The space in which something is viewed has a strong impact on personal reception for example seeing a display of original artefacts at a museum or at the original site encourages us to consider the context in which objects existed and were used. The online visual sphere on the other hand is a an impersonal form of reception appealing to the masses in an easily accessible easily understood form. Thus I wonder does the digital era have the possibility to discourage travel to museums and or other countries?

**Comment by JasNewtonRae** [comment]
I found the Clark article extremely interesting. Through living so much of our lives through the internet and digital appliances we often forget just /how/ vulnerable content created digitally is. Learning that there are these projects set up in order to capture these pieces of work that would otherwise dissapear is extremely pleasant. The fact that the Collection contains 200 websites is also extremely interesting and proves the large extent of this preservation work.

The Muenster article was interesting and I found the distinctions between digital heritage and digital humanities helpful.

**Comment by molmay** [comment]
Clark’s paper answered a key question that I had about the British Library’s Interactive Narratives collection - namely what happens to archived works if/when the programmes used to develop or host them become obsolete. I knew that we are able to visit ‘old’ versions of websites etc as they appeared at certain points in the past, but had no idea that some software could also emulate the use of now defunct programmes within older browsers such as Flash etc. I do wonder if such programmes are able to replicate all defunct browsers/programmes or if they can only replicate specific ones. My knowledge of this topic is fairly weak!

**Comment by Dasha-Barkova** [comment]
I think that Clark's article added to my understanding about digital heritage and the potential ways it could impact on the viewer/audience. I feel like the future of art offers the fascination of both media and new modes of access to old arts. Especially contemporary artists would appear to be at the forefront as we explore and expand our awareness, whether of radical social reforms or the rich heritage of the past.

**Comment by lettychardon** [comment]
> Clark’s paper answered a key question that I had about the British Library’s Interactive Narratives collection - namely what happens to archived works if/when the programmes used to develop or host them become obsolete. I knew that we are able to visit ‘old’ versions of websites etc as they appeared at certain points in the past, but had no idea that some software could also emulate the use of now defunct programmes within older browsers such as Flash etc. I do wonder if such programmes are able to replicate all defunct browsers/programmes or if they can only replicate specific ones. My knowledge of this topic is fairly weak!

I totally agree with this, Molly. Technology is advancing at such a rapid pace that by the time you finish something, a new and probably better version is available! I always love looking back at older versions of websites to see how much they have changed, but also how quickly everyone gets used to new versions, despite all the complaining right after updates!

Something that I've noticed is a recurring theme in many of the topics is the lack of conformity in the collection and display of digital data, and the Munster article really highlighted how unsustainable this approach is. I think it is important that there be a set of regulations (spoiler alert for the next set readings - like the London Charter) and that people actually adhere to them so as to facilitate the dissemination and shared use of findings.
